### ISSUE TITLE
HTTPS support

### ISSUE THREAD

**Issue by deadcoder0904** [freeCodeCamp/mail-for-good#302]
Currently I am using it for [my site](https://akshaykadam.me) & when I submit it to my HTTP URL, it gives me this warning

```
The information you have entered on this page will be sent 
over an insecure connection and could be read by a third party.

Are you sure you want to send this information?
```

So there should be a guide to add direct HTTPS support. I am currently trying LetsEncrypt or AWS Certificate Manager but finding it tedious unfortunately :(

**Comment by zhakkarn** [comment]
Yes there should be. Yes mail for good is stalling. Yes mail for good is not declared released yet. Yes it's not going to be declared released soon unless there is an unbelievable influx of contributors and a stupidly efficient way of dealing with them all without friction.

**Comment by deadcoder0904** [comment]
LOL now I know why the mail was for 😂

**Comment by avid88** [comment]
I'm very sorry to hear that mail-for-good is not going to continue to develop,
However, I found a new project that also using SES to send emails, for now it has no UI but maybe it can provide some value, I am using it.

**Comment by avid88** [comment]
I'm very sorry to hear that mail-for-good is not going to continue to develop,
However, I found a new project that also using SES to send emails, for now it has no UI but maybe it can provide some value, I am using it.
